ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -96,7 +96,7 @@ spec:
                 - containerPort: 10001
                   name: client
                 - containerPort: 8000
-                  name: serve
+                  name: serving
```

Comment:
maybe `serve-http-endpoint` ?

Reply:
What was wrong with serve? (serve as in Ray Serve)